roles/foreman/tasks/main.yaml

```
@@ -115,7 +123,8 @@
 
 - name: Wait for Foreman service to be accessible
   ansible.builtin.uri:
-    url: 'http://{{ ansible_hostname }}:3000/api/v2/ping'
+    url: '{{ foreman_url }}/api/v2/ping'
+    validate_certs: '{{ ca_certificate }}'
```

With that, I too get "The conditional check 'foreman_status.status == 200' failed. The error was: error while evaluating conditional (foreman_status.status == 200): 'dict object' has no attribute 'status'. 'dict object' has no attribute 'status'"
But not with:
```suggestion
    ca_path: '{{ ca_certificate }}'
```